snapcraft/tests/test_plugin_rust.py

```diff
@@ -64,15 +63,19 @@ def test_schema(self):
     def test_build(self, run_mock):
         plugin = rust.RustPlugin('test-part', self.options,
                                  self.project_options)
+        plugin.options.rust_features = ['conditional-compilation']
```

Comment:
I would prefer this to be in a separate test, to keep each unit test small and testing a single thing.